### PR TITLE
Diag indices

### LINKED ISSUE #9455: diag_indices
Numpy Frontend Indexing Routines #1536 Todolist item.

### PR DESCRIPTION
**Pull request by seancraven**
Close #9455

### CODE REVIEW

**Comment by seancraven** [comment]
This is not on the to-do list, however, a similar function diag_indices_from is. Their implementation is complimentary.

File: ivy/functional/frontends/numpy/indexing_routines/indexing_like_operations.py [hunk 1]
```diff
@@ -1,7 +1,14 @@
 import ivy
 from ivy.functional.frontends.numpy.func_wrapper import to_ivy_arrays_and_back
+from ivy.functional.frontends.numpy.func_wrapper import outputs_to_numpy_arrays
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/numpy/indexing_routines/indexing_like_operations.py:3]: This can be imported along with the wrapper above

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @seancraven 
Thanks for contributing to ivy😊
Also, thank you for bringing this function to our attention. Below are a few changes and your PR will get merged🙂

Thanks

**Comment by seancraven** [comment]
Thanks very much for the help!

**Review by hirwa-nshuti — APPROVED** [review]
Looks good to me🙂

Thanks for contributing

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Fix the conflicts and I'll get this merged very soon

**Review by hirwa-nshuti — APPROVED** [review]
Looks good now😊

**Pull request merged**